backend/controllers/autoCompleteController.js

```diff
@@ -0,0 +1,145 @@
+const { User, Tag, University, Department, Title, Interest } = require('../model/db');
+
+addTitle = async function (req, res) {
+    title=req.body.title
+    console.log("title: ", title)
+
+    try {
+        addedTitle = await Title.create({title: title})
+        res.status(200).send({message: "Title is created", title: addedTitle.title})
+    } catch (error) {
+        res.status(500).send(error)
+    }
+}
+
+addDepartment = async function (req, res) {
+    department=req.body.department
+    console.log("department: ", department)
+
+    try {
+        addedDepartment = await Department.create({department: department})
+        res.status(200).send({message: "Department is created", department: addedDepartment.department})
+    } catch (error) {
+        res.status(500).send(error)
+    }
+}
+
+addTag = async function (req, res) {
+    tag=req.body.tag
+    console.log("tag: ", tag)
+
+    try {
+        addedTag = await Tag.create({tag: tag})
+        res.status(200).send({message: "Tag is created", tag: addedTag.tag})
+    } catch (error) {
+        res.status(500).send(error)
+    }
+}
+
+addUniversity = async function (req, res) {
+    university=req.body.university
+    console.log("university: ", university)
+
+    try {
+        addedUniversity = await University.create({university: university})
+        res.status(200).send({message: "University is created", university: addedUniversity.university})
+    } catch (error) {
+        res.status(500).send(error)
+    }
+}
+
+addInterest = async function (req, res) {
+    interest=req.body.interest
+    console.log("interest: ", interest)
+
+    try {
+        addedInterest = await Interest.create({interest: interest})
+        res.status(200).send({message: "Interest is created", interest: addedInterest.interest})
+    } catch (error) {
+        res.status(500).send(error)
+    }
+}
+
+getTitles = async function (req, res) {
+    try {
+        titles = await Title.findAll({attributes: ['title']})
+        var array=[]
+        
+        for(var i in titles)
```

Comment:
pls fix that, we don't need arrays. that would make the code significantly better.

Reply:
I talked with the frontend and they prefered the output as array.

Reply:
Output is already an array

Reply:
No, the original output format is like this:

![image](https://user-images.githubusercontent.com/44067394/102550849-9f4d8e00-40cf-11eb-8e23-01a6ff9dd5d2.png)

However front end team preferred an output like this.

![image](https://user-images.githubusercontent.com/44067394/102550895-b5f3e500-40cf-11eb-81bb-6ddc3283f38b.png)

Reply:
Ok then, thanks